pkg/network/usm/ebpf_main.go

```diff
@@ -422,3 +481,71 @@ func (e *ebpfProgram) dumpMapsHandler(_ *manager.Manager, mapName string, curren
 	}
 	return output.String()
 }
+
+func (e *ebpfProgram) getProtocolStats() map[protocols.ProtocolType]interface{} {
+	ret := make(map[protocols.ProtocolType]interface{})
+
+	for _, protocol := range e.enabledProtocols {
+		ps := protocol.GetStats()
+		if ps != nil {
+			ret[ps.Type] = ps.Stats
+		}
+	}
+
+	return ret
+}
+
```

Comment:
Please add documentation for this function as you did in the commit, I found the information in the commit to be vary useful do understand the purpose of the function